vector/src/main/java/im/vector/app/features/home/room/detail/timeline/render/ProcessBodyOfReplyToEventUseCase.kt

```diff
@@ -100,10 +103,23 @@ class ProcessBodyOfReplyToEventUseCase @Inject constructor(
                         repliedToEvent.isPollEnd() -> stringProvider.getString(R.string.message_reply_to_sender_ended_poll)
                         else -> ""
                     }
+                    val repliedText = when {
+                        repliedToEvent.isPollEnd() -> {
```

Comment:
I think we should add a new unit test to cover the end poll event case.

Reply:
Unit tests updated in [7683b83](https://github.com/vector-im/element-android/pull/7969/commits/7683b8325c3535af3b478d3759ac7e00431c05fd)